packages/react/src/Suspense.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import { render, screen, waitFor } from '@testing-library/react'
+import { Suspense } from './Suspense'
+
+const ms = 1000
+const TEXT = 'Child'
+const FALLBACK = 'Fallback'
+
+const Throw = () => {
+  throw new Promise((resolve) => resolve('resolved'))
+}
+
+describe('Suspense', () => {
+  let needThrow = true
+  beforeEach(() => {
+    needThrow = true
+  })
+  const ThrowDuring = (props: { ms: number }) => {
```

Comment:
I think it may cause misunderstanding that the word `props` is used for react components, so why don't you modify it as follows?

```suggestion
  const ThrowDuring = (params: { ms: number }) => {
```

Reply:
<img width="566" alt="image" src="https://github.com/suspensive/react/assets/61593290/2212f3bf-93c2-4248-8875-3c4ff5ccb76f">

ThrowDuring should be used as a react component. If ThrowDuring is just used as a function, an error like the screenshot above may occur. So I think the name props is correct.

If you want to check it yourself, you can see what happens if you fix it like params and run ThrowDuring as a function instead of ThrowDuring react component. I don't think ThrowDuring should be used as a function because Suspense can catch the child component's pending promise.

Therefore, I think it should be used like ReactComponent with props rather than params.

Reply:
oh I totally understood by reading your cool comment 👍